index.js

```diff
@@ -12,24 +12,9 @@ module.exports = (robot) => {
     interval: 60 * 60 * 1000 // 1 hour
   });
 
-  robot.on('issue_comment.created', async context => {
-    if (
-      whitelistedAccounts.includes(context.repo().owner.toLowerCase()) &&
-      context.isBot === false) {
-      const userName = context.payload.comment.user.login;
-      if (pullRequestAuthor === userName) {
-        apiForSheetsModule.apiForSheets(userName, context, false);
-      }
-    }
-  });
-
   robot.on('schedule.repository', async context => {
-    if (
-      whitelistedAccounts.includes(context.repo().owner.toLowerCase()) &&
-      context.isBot === false) {
-      const userName = context.payload.pull_request.user.login;
-      pullRequestAuthor = userName;
-      apiForSheetsModule.apiForSheets(userName, context, true);
+    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {

```

Comment:
Just a note, I don't think I understand what this is doing. Would you mind explaining? I'm not sure if a comment might be helpful here.

Reply:
Done, thanks!